README.md

```diff
@@ -49,12 +48,92 @@ make build_ext
 make install
 ```
 
-## Notes
 
-Some test cases from w3lib have been modified to conform to the way Chromium GURL parses
-urls. In detail, url paths are usually lowercased since GURL is the urlparser of a browser.
+## Available `Make` commands
+
+Make commands create a shorter way to type commands while developing :)
+
+`make clean`
+
+This will clean the build dir and the files that are generated when running `build_ext` command
+
+`make test`
+
+This will run all the tests found in the `/tests` folder
+
+`make build_ext`
+
+This will run the command `python setup.py build_ext --inplace`, which builds Cython code
+for this project.
+
+`make sdist`
+
+This will run `python setup.py sdist` command on this project.
+
+`make install`
+
+This will run `python setup.py install` command on this project.
+
+`make develop`
+
+This will run `python setup.py develop` command on this project.
+
+`perf`
```

Comment:
Should be `make perf`?

Reply:
you are right! I will change all of these based on your suggestions 😄